ui/app/AppLayouts/Communities/panels/OverviewSettingsFooter.qml

```diff
@@ -29,9 +28,7 @@ Control {
         property string paragraphTitle
         property string paragraphSubtitle
         property string primaryButtonText
-        property string primaryButtonIcon
         property string secondaryButtonText
```

Comment:
Is this still necessary?

Reply:
Yeah, not needed, the button is always the same